vcd/resource_vcd_nsxv_firewall_rule.go

```diff
@@ -593,22 +591,22 @@ func getEndpointData(endpoint types.EdgeFirewallEndpoint, edge *govcd.EdgeGatewa
 	var (
 		endpointNetworks []string
 		endpointVMs      []string
-		// TODO uncomment when IP sets and Security groups are supported
-		// endpointIpSets         []string
+		endpointIpSets   []string
+		// TODO uncomment when Security groups are supported
 		// endpointSecurityGroups []string
 	)
 
 	for _, groupingObject := range endpoint.GroupingObjectIds {
 		idSplit := strings.Split(groupingObject, ":")
 		idLen := len(idSplit)
 		// TODO uncomment when IP sets and Security groups are supported
```

Comment:
Maybe it's just how GitHub shows things, but this comment looks obsolete now.

Reply:
I removed them. There is still one more comment related to security groups below.